### DISCUSSION TITLE
Markdown not rendered correctly in app description

### DISCUSSION THREAD

**Discussion by ktobah** [fastapi/fastapi#6545] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi.openapi.utils import get_openapi
from fastapi.openapi.docs import get_redoc_html, get_swagger_ui_html, get_swagger_ui_oauth2_redirect_html
from fastapi.staticfiles import StaticFiles


description = """
    ## Introduction

    An intro

    ## Classification

    The classification API ...

    * **Model**: some text.

    """

app = FastAPI(title="My Services", docs_url=None, redoc_url=None)

app.mount("/static", StaticFiles(directory="/app/app/static"), name="static")


def custom_openapi():
    if app.openapi_schema:
        return app.openapi_schema
    openapi_schema = get_openapi(
        title="My Services",
        version="1.0",
        description=description,
        routes=app.routes,
    )
    
    app.openapi_schema = openapi_schema
    return app.openapi_schema


app.openapi = custom_openapi


# The following configurations are added to allow to serve the docs Javascript and CSS files locally
# Therefore, the documentation can be visualized without the need for an internet connection.
@app.get("/docs", include_in_schema=False)
async def custom_swagger_ui_html():
    return get_swagger_ui_html(
        openapi_url=app.openapi_url,
        title=app.title + " - Swagger UI",
        oauth2_redirect_url=app.swagger_ui_oauth2_redirect_url,
        swagger_js_url="/static/swagger-ui-bundle.js",
        swagger_css_url="/static/swagger-ui.css",
        swagger_favicon_url="/static/favicon.ico"
    )


@app.get(app.swagger_ui_oauth2_redirect_url, include_in_schema=False)
async def swagger_ui_redirect():
    return get_swagger_ui_oauth2_redirect_html()


@app.get("/", include_in_schema=False)
async def redoc_html():
    return get_redoc_html(
        openapi_url=app.openapi_url,
        title=app.title + " - ReDoc",
        redoc_js_url="/static/redoc.standalone.js",
        redoc_favicon_url="/static/favicon.ico"
    )
```

### Description

I am trying to visualize the redoc documentation at "/" and following the tutorial for serving the static files locally. Everything works fine except the description of the app. It is supposed to render the markdown content but it doesn't (see the picture). I also tried to include the description in `app = FastAPI(...)` and call `description = app.description` in `custom_openapi()`, but it didn't work.

![image](https://user-images.githubusercontent.com/3786785/129800669-d4cdca13-65ac-4b1a-bb2e-b5b9231f66c5.png)

### Operating System

Windows

### Operating System Details

Docker is used to serve the app.

### FastAPI Version

0.68.0

### Python Version

3.8

### Additional Context

My app is complex, I use routes and add them using app.include_router(). I just posted a minimum code for reference.

I use custom_openapi() to supply a custom logo. I just omitted such information from the code snippet above.

**Comment by anthonycepeda** [comment]
Hi @ktobah , the problem is that it cannot be a Docstring but a regular string:
```python
description = "## Introduction\nAn intro\n## Classification\nThe classification API ...\n* **Model**: some text.\n"
```

![image](https://user-images.githubusercontent.com/72019805/131716749-041f27d3-6952-4e5c-aba8-9d3f8e765bf8.png)


Thanks

**Comment by ktobah** [comment]
Thank you very much. That solved the problem.

**Comment by daroczig** [comment, marked as answer]
Using docstring works, but you need to remove the whitespace on the left, so wrap it e.g. inside of [textwrap.dedent](http://docs.python.org/library/textwrap.html#textwrap.dedent), something like:


```py
description = """
    ## Introduction

    An intro

    ## Classification

    The classification API ...

    * **Model**: some text.

    """

app = FastAPI(title="My Services", description=textwrap.dedent(description))
```
